### PR TITLE
Add a way to add some documents

### PR DESCRIPTION
**Pull request by CPatchane**
Screenshot 2019-04-09 at 14 23 18
It was used to test and debug some services triggered by the Stack on documents CREATED events.

Since the app must have write permissions to doctypes, the select here is based on the sandbox manifest to get all allowed doctypes
To handle CozyClient (v2 here), the schema is also computed from the app manifest but it still doesn't work :/ :
Screenshot 2019-04-09 at 14 24 48

Also update cozy-ui and cozy-client to use the latests versions.

### CODE REVIEW

File: src/components/Documents/AddButton.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+    if (typeof onComplete === 'function') onComplete(response)
+  }
+
+  render() {
+    const { doctype, attributes, ...forwardProps } = this.props
+    const { v2 } = this.state
+    delete forwardProps.onComplete
+    return (
+      <div>
```
Comment (gregorylegarec) [inline src/components/Documents/AddButton.jsx]: Why not 
```jsx
return (
 <Consumer>
   { client => (<div>

	//..

    </div>)}
</Consumer>
```
?

File: src/components/Documents/AddButton.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  handleChange(ev) {
+    this.setState({ v2: ev.target.checked })
+  }
+
+  async create() {
+    const { client, doctype, attributes, onComplete } = this.props
+    const { v2 } = this.state
+    let response
+    if (v2) {
+      response = await client.mutate(client.create(doctype, attributes))
```
Comment (gregorylegarec) [inline src/components/Documents/AddButton.jsx]: `client.mutate()` [is expecting a `mutationDefinition`](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L374). We should directly call [`client.create`](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L181).

I think it's too expensive to maintain two clients here. Let's use CozyClient.

**Comment by nono** [comment]
Ping: what is the status of this PR?

**Comment by nono** [comment]
Ping @CPatchane and @gregorylegarec

**Comment by nono** [comment]
Ping @gregorylegarec and @CPatchane

**Comment by nono** [comment]
Ping @gregorylegarec

**Comment by gregorylegarec** [comment]
I think we can close it.

**Pull request closed without merging**